catroid/src/main/java/org/catrobat/catroid/physics/content/bricks/TurnRightSpeedBrick.java

```diff
@@ -91,7 +91,8 @@ public View getView(Context context, int brickId, BaseAdapter baseAdapter) {
 	public View getPrototypeView(Context context) {
 		prototypeView = View.inflate(context, R.layout.brick_physics_turn_right_speed, null);
 		TextView textTurnRightSpeed = (TextView) prototypeView.findViewById(R.id.brick_turn_right_speed_edit_text);
-		textTurnRightSpeed.setText(String.valueOf(BrickValues.PHYSIC_TURN_DEGREES));
+		textTurnRightSpeed.setText(FormulaBrick.getStringFromFloatForBrickView(BrickValues
```

Comment:
please make the method in formulaBrick protected and remove the static. then FormulaBrick.getStringFromFloatForBrickView can become getStringFromFloatForBrickView...

for the one usage in InternToExternGenerator please refactor this method.